lib/gemoji.rb

```diff
@@ -1 +1,2 @@
 require 'emoji'
+Emoji.all
```

Comment:
Can this go to `emoji.rb` instead, so that `require gemoji` and `require emoji` (which are supposed to be equivalent) don't lead to different experiences?